lib/new_relic/agent/instrumentation/grpc/client/prepend.rb

```diff
@@ -26,7 +26,7 @@ def bidi_streamer(method, requests, marshal, unmarshal,
               metadata: {},
               &blk)
 
-              issue_request_with_tracing(method, requests, marshal, unmarshal,
+              issue_request_with_tracing(:bidi_streamer, method, requests, marshal, unmarshal,
```

Comment:
- For each instrumented gRPC server request handling method, report its name to
  `issue_request_with_tracing`, and then report that name as `grpc_type`
  (thanks, Node.js agent team for the idea)